service/gcs/prot/protocol.go

```diff
@@ -448,6 +453,12 @@ type ContainerGetProperties struct {
 	Query string
 }
 
+// ContainerUpdate is the message sent to update a container's resources
+type ContainerUpdate struct {
+	MessageBase
+	Resources string
```

Comment:
This looks like it's just [LinuxResources](https://pkg.go.dev/github.com/opencontainers/runtime-spec/specs-go#LinuxResources), should we use that type instead?

Reply:
I was running into issues with unmarshalling the `ContainerUpdate` struct when doing that, when I googled for it, people mentioned it was because there was an embedded type that needed to be individually unmarshalled. In that case, I figured it would be easier to keep this a string since runc reads from stdin (so reads in a string) anyways. I can look into it again though to see if I was maybe just doing something wrong.